src/shared/components/QueryBuilder/domain/helpers/filtersToQuery.ts

```diff
@@ -6,9 +6,9 @@ export function filtersToQuery(query: string, filters: Filters) {
     .map((filter) => {
       const { attribute, operator, value } = filter as CompleteFilter;
 
-      if (operator.value === OperatorKind.in) {
-        return `${attribute.value}=~"${value.value}"`;
-      }
+      // if (operator.value === OperatorKind.in) {
```

Comment:
Not strictly related to this PR, but if we execute this code, the "in" operator will be rendered in the filters. This cannot be because [it's not included in the operators dropdown options](https://github.com/grafana/explore-profiles/blob/main/src/shared/components/QueryBuilder/infrastructure/operatorsRepository.ts#L12).